lib/gruf/server.rb

```diff
@@ -89,31 +86,55 @@ def server
     # Start the gRPC server
     #
     # :nocov:
-    def start!
-      update_proc_title(:starting)
+    def start
```

Comment:
But in any case, I need changes in this part so as not to cause blocking of the main thread.